.github/workflows/ci.yml

```diff
@@ -34,3 +52,16 @@ jobs:
 
       - name: Run Tests
         run: v test .
+
+  cffconvert:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout ${{ github.repository }}
+        uses: actions/checkout@v3
+        with:
+          persist-credentials: false
+
+      - name: Lint CITATION.cff
+        uses: docker://citationcff/cffconvert:latest
+        with:
+          args: --validate
```

Comment:
This is the linter for the CITATION.cff.  It is a Docker image around the Python 3 CLI defined here:  https://github.com/citation-file-format/cff-converter-python.  The Docker image usually succeeds within about 10 seconds, so many projects I configured CFF for prefer to always run the linter.  There is also an Action which is a wrapper around the Docker image but I assume calling the Docker image directly might be the more convenient way.  The Docker tag `latest` will always pull the latest image, so it is maintenance-free.

Reply:
Nice! I like that it's maintenance free lol